f8a_report/stack_report_main.py

```diff
@@ -33,8 +44,60 @@ def main():
         logger.exception("Error Generating v2 report")
         raise e
 
-    return response
+    return missing_latest_nodes
 
 
 if __name__ == '__main__':
-    main()
+    missing_latest_nodes = main()
+
+    """
+    Initializing Selinon queues and celery workers.
+
+    If report has packages which are missing any version from Db, ingestion flow
+    for such packages will be triggered to ingest requested version into Graph DB
+    using Jobs API.
+    """
+    if _INVOKE_API_WORKERS == "1":
```

Comment:
anything related to workers should not be exposed outside. Have all this within the ingestion API

Reply:
+1, you should only call the endpoint from here, everything else should be handled in the api.